frontend/src/app/main/ui/workspace/snap_distances.cljs

```diff
@@ -146,13 +163,14 @@
         ;; Gets the distance to the current selection
         lt-distances (->> lt-shapes (map distance-to-selrect) (filter pos?) (into #{}))
```

Comment:
did't see before, but here you can share the same transducer for two operations and avoid create 4 seqs